edb/lib/ext/auth.edgeql

```diff
@@ -65,6 +65,13 @@ CREATE EXTENSION PACKAGE auth VERSION '1.0' {
         create required property password_hash: std::str;
     };
 
+    create type ext::auth::PKCE extending ext::auth::Auditable {
+        create required property challenge: std::str {
```

Comment:
We will never remove these objects?

Reply:
We delete these at the end of the flow once you've completed.

There is an additional issue with "abandoned" PKCE flows that would pile up without GC. I had a couple of ideas here:

1. Maybe we already have some kind of cron-like/job-queue that we can schedule a cleanup with?
2. We could remove all outdated challenges at the beginning of each new challenge.

Reply:
We can remove all outdated ones on server startup for now.